plugins/inputs/http_listener_v2/http_listener_v2.go

```diff
@@ -188,11 +207,17 @@ func (h *HTTPListenerV2) createHTTPServer() *http.Server {
 
 // Stop cleans up all resources
 func (h *HTTPListenerV2) Stop() {
+	if h.server != nil {
+		ctx, cancel := context.WithTimeout(context.Background(), serverShutdownTimeout)
+		defer cancel()
+		h.server.Shutdown(ctx)
```

Comment:
We could theoretically not look at the error returned from `.Shutdown` (as below with `h.listener.Close()`) but then we'd have to add a linter comment to prevent it complaining about this: `//nolint:errcheck,revive`


```
http_listener_v2.go:213:20  errcheck  Error return value of `h.server.Shutdown` is not checked
```

Reply:
I've decided to actually check it and log the error.